package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
```

Comment:
If the format is "2", do we still need the distinction between build depend and exec depend?

Reply:
depend=build_depend+build_export_depend+exec_depend. All the dependencies are not exported, so than depend would be too much in my opinion.